src/NServiceBus.AcceptanceTests/NServiceBus.AcceptanceTests.csproj

```diff
@@ -40,6 +40,11 @@
     <_PackageFiles Remove="**\obj\**\*.cs" />
     <_PackageFiles Remove="Core\**\*.cs" />
     <_PackageFiles Remove="AssemblyInfo.cs" />
+    <_PackageFiles Remove="Sagas\When_correlated_property_value_is_changed.cs" />
```

Comment:
I think this is done automatically by VS for some reason. should be removed though.